conf/conf.sh.dist

```diff
@@ -1,32 +0,0 @@
-#!/usr/bin/env bash
-
-## CUSTOM SQL - Important file used by the db_assembler.sh
-## Keep only the required variables (base sql files or updates, depending on the DB)
-
-## BASE SQL
-
-DB_AUTH_CUSTOM_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/auth/base/"
-)
-
-DB_CHARACTERS_CUSTOM_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/characters/base/"
-)
-
-DB_WORLD_CUSTOM_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/world/base/"
-)
-
-## UPDATES
-
-DB_AUTH_UPDATES_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/auth/updates/"
-)
-
-DB_CHARACTERS_UPDATES_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/characters/updates/"
-)
-
-DB_WORLD_UPDATES_PATHS+=(
-	"$MOD_SKELETON_ROOT/sql/world/updates/"
-)
```

Comment:
why you removed this? to use some other default confs?

Reply:
Apparently yes. I saw it in another module.
https://github.com/azerothcore/mod-weekend-xp
The idea is that it executes the SQL automatically.
In fact it did, I did not run any SQL.

Reply:
ok 👍 , I was thinking that this skeleton had the right directory structure for this, but apparently I was wrong